pkg/monitor/monitor_cmd/command.go

```diff
@@ -33,9 +34,11 @@ type TimelineOptions struct {
 	PodResourceFilename  string
 	TimelineType         string
 
-	LocatorMatchers []string
-	Namespaces      []string
-	OutputType      string
+	LocatorMatchers        []string
+	RemovedLocatorMatchers []string
```

Comment:
cruft

Reply:
I still use `RemovedLocatorMatchers` for inverse matchers in the `NotContainsAllParts` function.

Reply:
never has a value, right?

Reply:
cruft removed :-)